src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -28,84 +32,110 @@
  * floating around.
  */
 public class RobotMap {
-	public static class Port {
+	public static class Port { // TODO: Update ports to match those on Timón.
 		public static class HumanInput {
 			public static final int joystick = 0;
 			public static final int xboxController = 1;
 		}
-		
+
+		public static class Motors {
+			public static class CAN {
+				public static int leftDriveA = 1;
+				public static int leftDriveB = 2;
+				public static int rightDriveA = 3;
+				public static int rightDriveB = 4;
+			}
+		}
+
 		public static class PWM {
 			public static final int leftDriveMotor = 0;
 			public static final int rightDriveMotor = 1;
 			public static final int ballInnie = 2;
-			// public static final int trayInnie = 3; ----- PROJECT CURRENTLY PAUSED
 			public static final int flywheelLeftMotor = 4;
 			public static final int flywheelRightMotor = 5;
 			public static final int vomitElevator = 6;
 			public static final int vomitOuttakeRoller = 7;
 		}
-		
+
 		public static class CAN {
-			public static final int leftEncoder = 0x602;
-			public static final int rightEncoder = 0x603;
+			public static final int leftEncoder = 0x611;
+			public static final int rightEncoder = 0x612;
 			public static final int trayEncoder = 0x604;
 			public static final int flywheelEncoder = 0x605;
 			public static final int elevatorEncoder = 0x606;
 		}
-		
+
 		public static class CANMotor {}
-		
+
 		public static class PCM {}
-		
-		public static class Pneumatics {}
+
+		public static class Pneumatics {
+			public static int solenoidUp = 0;
+			public static int solenoidDown = 1;
+		}
 	}
-	
+
+	public static class Metrics {
+		public static final double WHEEL_PULSES_PER_REVOLUTION = 1024;
+	}
+
 	public static class Component {
+		public static class HumanInput {
```

Comment:
HumanInput class shouldn't exist. The values it contains should be renamed and stuck in the root level of the Component class.